docs/tutorial/01-getting-started.md

```diff
@@ -0,0 +1,367 @@
+- `packages/`: holds each smart contract's client and types as a separate package for the project's NPM workspace. These are built by Scaffold Stellar and we should not manually edit them. They'll be used by the frontend.
+- `.env`: is where we store environment variables that we'll be used by Scaffold Stellar commands.
+- `environments.toml`: This is the Scaffold Stellar secret sauce! This file is where we deterministically configure:
+  - our project's various _environments_, ...
+  - which _networks_ are used by each environment, ...
+  - all in service of which _contracts_ our project depends on in each of those environments.
+
+So how do all these pieces work together? Here's what Scaffold Stellar handles for you:
+
+1. Our `npm start` command runs `stellar scaffold watch --build-clients`
+2. Our `.env` file set an environment variable to say we're in our _development_ environment (`STELLAR_SCAFFOLD_ENV=development`)
+3. Scaffold Stellar looked to `environments.toml` for the development environment's configuration, which told it to:
+    - Start up a local Stellar network
+    - Create an account on the network
+    - Build the contracts
+    - Deploy them to the network
+    - Generate their clients for the frontend
+
+That's a lot of heavy lifting! Normally you'd have to do all this yourself, perhaps in a procedural script, but Scaffold Stellar does it for you. And it's deterministic, meaning you can always reproduce the same results from the same environment configuration. You set configuration values, specifying the desired starting state for your app, and Scaffold Stellar does all the work to get your app into that state.
+
+And we're just getting started! We haven't even looked at the application yet.
+
+## 🚀 Open the App
+
+The `npm start` command should have finished building the contracts by now so you can open the app in your browser. It should be running at Vite's default port, [http://localhost:5173](), and you should see the home page:
+
+```
+Welcome to your app!
+
+...
+
+<GuessTheNumber />
+Connect wallet to play the guessing game
+```
+
+In order to test out our deployed example contract, we'll need to connect to a wallet.
+
+### 💰 Connect a Wallet
+
+In the top right corner, you'll see a big "Connect" button. Click it. You need to have a [Wallet](https://stellar.org/learn/wallets-to-store-send-and-receive-lumens) in order to interact with the dApp. The modal that opened will show a few options if you don't have one already. We recommend using [Freighter](https://www.freighter.app/).
+
+Once it's installed, we need to connect it to our local network running in Docker. Open the extension, click the menu, and navigate to "Settings," then "Network." Click the "Add custom network" button and enter the following info:
```

Comment:
Outside the scope of this, but it would  be *great* if freighter could have these defaults.